src/fromager/overrides.py

```diff
@@ -108,12 +109,23 @@ def patches_for_requirement(
     override_name = pkgname_to_override_module(req.name)
     unversioned_patch_dir = patches_dir / override_name
     versioned_patch_dir = patches_dir / f"{override_name}-{version}"
+
+    unversioned_patch_files = list(unversioned_patch_dir.glob("*.patch"))
+    versioned_patch_files = list(versioned_patch_dir.glob("*.patch"))
+
+    # The list of files must exist to be joined to the global patch files
```

Comment:
For versioned vs. unversioned we use different directories. We follow that pattern elsewhere downstream with directories for the variant-specific parts. What's the benefit of introducing a new approach for these files?

Reply:
Not sure I get the question. I am only moving this `glob` here to allow adding files to the list before sorting it. The versioned and unversioned directories don't change, only the suffix we're looking for.

Reply:
I have suggested `patches_dir / override_name / variant / "*.patch"` and `patches_dir / f"{override_name}-{version}" / variant / "*.patch"`. This keeps the variant patches in a subfolder. IMHO that is less clutter and easier to understand.

Reply:
I like that. It also makes it less likely we would have some sort of issue parsing the path name.